lib/manageiq/api/common/action_dispatch-routing-option_redirect.rb

```diff
@@ -0,0 +1,23 @@
+module ManageIQ
```

Comment:
Minor, but I think this file should be renamed and/or the class renamed to match.  I can merge w/o this though.

Reply:
Renamed the file.